modules/custom/az_migration/migrations/migrate_plus.migration.az_paragraph_column_image.yml

```diff
@@ -0,0 +1,46 @@
+id: az_paragraph_column_image
+label: AZ Paragraph Column Image
+migration_group: az_migration
+migration_tags:
+  - Quickstart Content Migration
+status: true
+
+source:
+  plugin: az_paragraphs_item
+  bundle: uaqs_column_image
+
+destination:
+  plugin: 'entity_reference_revisions:paragraph'
+  default_bundle: az_text
+
+process:
+
+  temp_photo:
+    plugin: sub_process
+    source: field_uaqs_photo
+    process:
+       - 
+        plugin: migration_lookup
+        source: fid
+        migration: 
+          - az_media
+
+  field_az_text_area: 
+    plugin: paragraphs_column_image_field_merge
+    source: '@temp_photo'
+    caption: field_uaqs_caption_text
+    credit: field_uaqs_image_credit
```

Comment:
Just verifying this is _not_ modifying the caption or credit fields on the media entity, correct?

Reply:
> Just verifying this is _not_ modifying the caption or credit fields on the media entity, correct?

Looks that way to me.

Reply:
Yes, it is not. We are appending the values as in the mapping excel sheet.